pkgs/tools/system/rsyslog/default.nix

```diff
@@ -103,6 +104,10 @@ stdenv.mkDerivation rec {
     (mkFlag true                      "generate-man-pages")
   ];
 
+  passthru.tests = {
+    nixos-rsyslogd = nixosTests.rsyslogd;
```

Comment:
can we just `inherit (nixosTests) rsyslogd`?

Reply:
I prefer the attributes to be a bit more descriptive, especially when using tab completion.